content/en/docs/ops/integrations/spire/index.md

```diff
@@ -0,0 +1,385 @@
+
+   {{< text >}}
+   socket_path = "/run/secrets/workload-spiffe-uds/socket"
+   {{< /text >}}
+
+2. Share the SPIRE Agent socket with the pods within the node by leveraging the
+   [SPIFFE CSI Driver](https://github.com/spiffe/spiffe-csi).
+
+3. See [Install Istio](#install-istio) to configure Istio to integrate with the SPIFFE CSI Driver.
+
+## Install Istio
+
+1. [Download the latest Istio release](https://istio.io/latest/docs/setup/getting-started/#download).
+
+to access the SPIRE Agent socket shared by the [SPIFFE CSI Driver](https://github.com/spiffe/spiffe-csi). 
+
+2. After successfully deploying SPIRE into your environment, install Istio with custom patches for istio-ingressgateways/egressgateways as well as the istio-proxy.
+
+   {{< text bash >}}
+   $ istioctl install --skip-confirmation -f - <<EOF
+   apiVersion: install.istio.io/v1alpha1
+   kind: IstioOperator
+   metadata:
+     namespace: istio-system
+   spec:
+     profile: default
+     meshConfig:
+       trustDomain: example.org
+     values:
+       global:
+       # This is used to customize the sidecar template
+       sidecarInjectorWebhook:
+         templates:
+           spire: |
+             spec:
+               containers:
+               - name: istio-proxy
+                 volumeMounts:
+                 - name: workload-socket
+                   mountPath: /run/secrets/workload-spiffe-uds
+                   readOnly: true
+               volumes:
+                 - name: workload-socket
+                   csi:
+                     driver: "csi.spiffe.io"
+     components:
+       ingressGateways:
+         - name: istio-ingressgateway
+           enabled: true
+           label:
+             istio: ingressgateway
+           k8s:
+             overlays:
+               - apiVersion: apps/v1
+                 kind: Deployment
+                 name: istio-ingressgateway
+                 patches:
+                   - path: spec.template.spec.volumes.[name:workload-socket]
+                     value:
+                       name: workload-socket
+                       csi:
+                         driver: "csi.spiffe.io"
+                   - path: spec.template.spec.containers.[name:istio-proxy].volumeMounts.[name:workload-socket]
+                     value:
+                       name: workload-socket
+                       mountPath: "/run/secrets/workload-spiffe-uds"
+                       readOnly: true
+   EOF
+   {{< /text >}}
+
+3. Use [Automatic sidecar injection](https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection)
+to inject automatically the spire template, in the configuration above, into new workloads pods.
+
+This will share the spiffe-csi-driver with the Ingress Gateway and the sidecars that are going
+to be automatically injected on workload pods, granting them access to the SPIRE Agent's UNIX Domain Socket.
+
+* Check Ingressgateway pod state:

```

Comment:
Following [Istio docs on Ingress Gateways](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/), I think that if we are talking about the concept, it should be named like this -two words-, if we talk about the specific deployed component it should be `ingressgateway`, @sanderson042 thoughts?
```suggestion
* Check Ingress Gateway pod state:

```